shopfloor/models/priority_postpone_mixin.py

```diff
@@ -1,40 +1,33 @@
 # Copyright 2020 Camptocamp SA (http://www.camptocamp.com)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class PriorityPostponeMixin(models.AbstractModel):
     _name = "shopfloor.priority.postpone.mixin"
     _description = "Adds shopfloor priority/postpone fields"
 
-    # shopfloor_priority is set to this value when postponed
-    # consider it as the max value for priority
-    _SF_PRIORITY_POSTPONED = 9999
     _SF_PRIORITY_DEFAULT = 10
 
     shopfloor_priority = fields.Integer(
         default=lambda self: self._SF_PRIORITY_DEFAULT,
         copy=False,
         help="Technical field. Overrides operation priority in barcode scenario.",
     )
-
     shopfloor_postponed = fields.Boolean(
```

Comment:
@simahawk we are sending the value of this field to the client in the cluster picking scenario: https://github.com/OCA/wms/blob/13.0/shopfloor/services/cluster_picking.py#L395
With this PR this field will always return `True` as soon as it has been postponed at least one time, is it an issue? (I suppose no...).

Reply:
that's fine. It only sets a class on the line block to highlight it if required when the line is postponed.